src/common/logger/tests/logger_test.cpp

```diff
@@ -93,10 +65,8 @@ TEST_F(LoggerConstructorTest, WindowsLoggerConstructor)
     auto win_sink = std::dynamic_pointer_cast<spdlog::sinks::win_eventlog_sink_st>(sinks[0]);
     EXPECT_NE(win_sink, nullptr);
 }
-#endif // _WIN32
-
-#ifdef __APPLE__
-TEST_F(LoggerConstructorTest, MacOSLoggerConstructor)
+#else // _WIN32
```

Comment:
I think that it would be clear this line as

```
#else // __linux__ || __APPLE__
```

Like "...else, linux or apple are defined."

Reply:
True. Done.